Functions.Templates/Templates-v2/DurableFunctions-Python/template.json

```diff
@@ -0,0 +1,176 @@
+{
+    "author": "Varad Meru (@vrdmr)",
+    "name": "Blob trigger",
+    "description": "$BlobTrigger_description",
```

Comment:
```suggestion
    "description": "$OrchestrationTrigger_description",
```

Reply:
Also please go to `/Resources/Resources.resx` and create this variable.
For example, for `BlobTrigger_description`, that variable is defined like this: https://github.com/Azure/azure-functions-templates/blob/ceca1157044975bad1706ea17fb070b1bbb868bb/Functions.Templates/Resources/Resources.resx#L828-L830